.github/workflows/jira-sync.yml

```
@@ -0,0 +1,21 @@
+name: Sync with JIRA
+
+on:
+  workflow_dispatch: 
+  issues: 
+    types:
+      - opened
+      - edited
+      - transferred
+      - reopened
+
+jobs:
+  sync-to-jira:
+    uses: telicent-oss/shared-workflows/.github/workflows/jira-sync.yml@jira-sync
```

```suggestion
    uses: telicent-oss/shared-workflows/.github/workflows/jira-sync.yml@main
```
Should target the `main` branch version of the workflow

---

Thanks Rob